generation/maisi/README.md

```diff
@@ -30,16 +30,57 @@ MAISI is based on the following papers:
 ### 1. Installation
 Please refer to the [Installation of MONAI Generative Model](../README.md).
```

Comment:
Hi @Can-Zhao , since it only depends on monai (`monai-generative` is no longer needed), shall we remove the installation note here?

Reply:
Done. Thanks